### DISCUSSION TITLE
Lower resolution of overlayed image when exported to html

### DISCUSSION THREAD

**Discussion by zxdawn** [opengeos/leafmap#555] [Q&A] [answered]
I see the resolution of overlayed png file in html exported by leafmap is lower than that exported by folium:

folium (zoom in): 
<img width="746" alt="image" src="https://github.com/opengeos/leafmap/assets/30388627/825899a8-553c-4af6-bc84-23cc301324f6">

leafmap (zoom in):
<img width="951" alt="image" src="https://github.com/opengeos/leafmap/assets/30388627/96991a52-9a7b-451d-a49b-b66df89e0115">

Any parameter in leafmap to control the resolution?

BTW, the HTML filesize is doubled with leafmap .... I don't know why the larger size leads to lower resolution.

**Comment by giswqs** [comment]
What do you mean by 'overlayed png file'?

**Reply by zxdawn** [reply]
Like this: https://leafmap.org/notebooks/33_image_overlay/

**Comment by giswqs** [comment, marked as answer]
Try https://leafmap.org/leafmap/#leafmap.leafmap.Map.add_image
